init-odkx-sync-endpoint.py

```diff
@@ -19,7 +19,7 @@ def run_interactive_config():
     env_file_location = os.path.join(os.path.dirname(__file__), "config", "https.env")
 
     try:
-        domain, email = parse_env_file(env_file_location)
+        domain, email, stack_name = parse_env_file(env_file_location)
```

Comment:
For a user coming from a previous version, `stack_name` would be `None` here. Please add a check.